example-context/example-service/example-query/example-query-controller/pom.xml

```diff
@@ -36,6 +36,9 @@
                                 ${project.build.directory}/generated-sources/controller
                             </outputDirectory>
                             <basePackageName>uk.gov.justice.api</basePackageName>
+                            <generatorProperties>
```

Comment:
If we are enabling this then in the pom for example app (across all Service Components) then we will need to add mappings in the RAML files for the new transaction endpoint when you do a merge / rebase for master.